examples/actions/url-rewrite.mdx

```diff
@@ -51,15 +51,15 @@ export const TemplatingExamples = () => (
 			}}
 		/>
 		<ConfigExample
-			snippetText={"variable replacement with comparisons"}
+			snippetText={"variable replacement with comparisons on partial path"}
 			config={{
 				actions: [
 					{
 						type: "url-rewrite",
 						config: {
 							format: "ngrok",
-							from: "/v0/api/when-will-then-be-now?not-now=${conn.start_ts > time.now}",
-							to: "/v1/api/soon?now=${conn.start_ts < time.now}",
+							from: "?not-now=${conn.ts.start > time.now}",
+							to: "?now=${conn.ts.start < time.now}",
```

Comment:
think these vars in docs need to be updated still too

Reply:
https://github.com/ngrok/ngrok-docs/blob/main/docs/http/traffic-policy/expressions/variables.mdx?plain=1#L15

Reply:
and here https://github.com/ngrok/ngrok-docs/blob/main/docs/tcp/traffic-policy/expressions/variables.mdx?plain=1#L14

Reply:
and here https://github.com/ngrok/ngrok-docs/blob/main/docs/tls/traffic-policy/expressions/variables.mdx?plain=1#L14
(these should probably be updated to be component)

Reply:
I think Ben is doing these and didn't want to collide with him @benjaminchan Is that still true?

Reply:
eh, I'll just do it, #yolo